packages/rs-drive-abci/src/execution/platform_events/identity_credit_withdrawal/update_broadcasted_withdrawal_transaction_statuses/v0/mod.rs

```diff
@@ -59,20 +63,39 @@ where
             )));
         };
 
-        let core_transactions = self.fetch_core_block_transactions(
-            last_synced_core_height,
-            block_execution_context
-                .block_state_info()
-                .core_chain_locked_height(),
-            platform_version,
-        )?;
-
         let broadcasted_withdrawal_documents = self.drive.fetch_withdrawal_documents_by_status(
             withdrawals_contract::WithdrawalStatus::BROADCASTED.into(),
             Some(transaction),
             platform_version,
         )?;
 
+        // Collecting only documents that have been updated
+        let transactions_to_check: Vec<Identifier> = broadcasted_withdrawal_documents
```

Comment:
Size is the same but semantically it's wrong. For Transaction ID we use dashcore::TxId